src/main/java/it/gov/pagopa/pu/worker/paymentsreporting/PaymentsReportingDaoImpl.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.pu.worker.paymentsreporting;
+
+import it.gov.pagopa.payhub.activities.dao.PaymentsReportingDao;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import it.gov.pagopa.pu.worker.paymentsreporting.mapper.PaymentsReportingMapper;
+import it.gov.pagopa.pu.worker.paymentsreporting.model.PaymentsReporting;
+import it.gov.pagopa.pu.worker.paymentsreporting.service.PaymentsReportingService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PaymentsReportingDaoImpl implements PaymentsReportingDao {
+    private final PaymentsReportingService paymentsReportingService;
```

Comment:
This is a DAO, it should already represent the persistence layer, it should not to return into the service layer, it could call directly the DAO

This PaymentsReportingService is not necessary